src/app/event-bus.ts

```diff
@@ -0,0 +1,5 @@
+import Vue                from 'vue';
```

Comment:
we already have an event bus: https://github.com/devCrossNet/vue-starter/blob/master/src/app/shared/services/EventBus.ts